src/features/swap/index.tsx

```diff
@@ -61,6 +50,22 @@ const SwapWidget = ({ sell }: Params) => {
   const wallet = useWallet()
   const { isConsentAccepted, onAccept } = useSwapConsent()
 
+  const appData: SafeAppData = useMemo(
+    () => ({
+      id: 1,
+      url: 'https://app.safe.global',
+      name: `${capitalize(tradeType.toLowerCase())} order`,
```

Comment:
No, I don't think this is a good idea as the name and url is submitted as origin to cgw:
<img width="953" alt="grafik" src="https://github.com/safe-global/safe-wallet-web/assets/693770/f07c7333-2b83-465d-a4fe-734a53e4ee13">

also the label for GTM is also dependent on it. 

@schmanu what do you think about this?

Reply:
We could also keep the name the same here and add an exception to the `AppTitle` component.

Reply:
I kept the `Safe Swap` name in the config and handle it in `AppTitle` now